html/css/styles.css

```diff
@@ -838,6 +838,13 @@ a.interface-upup:hover, a.interface-updown:hover, a.interface-admindown:hover {
   margin: 8px;
 }
 
+.infobox pre {
```

Comment:
This css selector is a little generic.  I'd prefer you make it more specific (with a class for example) or just put the style in the html code.